wagtailio/project_styleguide/templates/patterns/components/streamfields/cards/logo_card_block.html

```diff
@@ -12,9 +12,15 @@ <h4 class="logo-card__heading teaser-heading">{{ value.heading }}</h4>
     <p class="logo-card__description">{{ value.description }}</p>
 {% endif %}
 
-{% if value.logo %}
+{% if value.author and value.logo %}
```

Comment:
`value.logo` for the author avatar doesn't feel right. Maybe it's a wider discussion of if this should be it's own template however they are very similar apart from the logo / avatar

Reply:
I think this may need [changing in BE too](https://github.com/wagtail/wagtail.org/blob/34-blog-page/wagtailio/core/blocks.py#L274) so i'll raise with them to see what they suggest.